### ISSUE TITLE
Use the value of $SSH_CLIENT

### ISSUE THREAD

**Issue by danielcherubini** [lemonade-command/lemonade#40]
Hi,

Great package, one thing I think you could do, which would make it work without configuration, is to use the $SSH_CLIENT environment variable, if it's populated, then get the IP address from there, otherwise use the rest of your logic.

OR 

use what's in the config, then if that's empty use the SSH_CLIENT env

thoughts?

**Comment by marco-gallegos** [comment]
this sounds very cool

**Comment by vaqxai** [comment]
yes please, i was just meaning to ask about that functionality. we can't put an env var in the toml config file
